src/applications/ezr/containers/App.jsx

```diff
@@ -77,16 +98,15 @@ const App = props => {
   // Add Datadog UX monitoring to the application
   useBrowserMonitoring();
 
-  return isAppLoading || !isProdEnabled ? (
-    <va-loading-indicator
-      message={content['load-app']}
-      class="vads-u-margin-y--4"
-      set-focus
-    />
+  return isAuthOnlyEnabled && !environment.isLocalhost() ? (
```

Comment:
I added the `environment.isLocalhost()` check in order to avoid local development issues (without it, the app attempts to redirect the user to the verify path, but it breaks locally).